app/record/lists/list.js

```diff
@@ -33,7 +33,7 @@ class List extends BaseObject {
 
         const node = {};
 
-        node[type] = {};
+        node[type] = [];
 
         if (attributes) {
             node[type]["$attributes"] = attributes;
```

Comment:
this will be broken, no?

Reply:
I was also concerned from that but it works as expected